### DISCUSSION TITLE
Collaborate?

### DISCUSSION THREAD

**Discussion by shazz** [salem-ok/PySprint#55] [General]
Hey @salem-ok, thanks for inviting me :)
How would you like us to collaborate ? For each refactoring task I can create a branch then you'll review it and decide to merge it or not. Would it work for you ?

Tell me :)

**Comment by salem-ok** [comment]
Hey @shazz  welcome aboard :)
Yes let's work that way and see how it goes!

**Comment by shazz** [comment]
Thanks :)
I created a few PRs, let me know if it is ok for you!

**Reply by shazz** [reply]
@salem-ok, what about creating a test track dedicated to... test :) with all kinds of obstacles, gates, ramps.... possible?
